garage/tf/misc/tensor_utils.py

```diff
@@ -123,7 +123,14 @@ def pad_tensor_dict(tensor_dict, max_len):
     return ret
 
 
-class enclosing_scope(object):
+def normalized_inputs(inputs, instance=()):
```

Comment:
can you explain the purpose of this function?

Reply:
Transform the `inputs` to the same type as `instance`. I found many functions in tf.optimizers have `extra_inputs`,  so calling `inputs = normalized_inputs(inputs, extra_inputs)` transforms two into the same type.

Reply:
Transform the `inputs` to the same type as `instance`. I found many functions in tf.optimizers have `extra_inputs` but not in the same type, some have type list, some tuple. Calling `inputs = normalized_inputs(inputs, extra_inputs)` transforms two into the same type.